docs/Developers/Writing_Good_Commits.rst

```diff
@@ -173,52 +174,61 @@ Example:
 More Examples
 ~~~~~~~~~~~~~
 
-Example 1 (fixed bug):
+Example 1 (fixed bug and added enhacement):
+Pull request `#4217 <https://github.com/coala/coala/pull/4217>`_
 
 ::
 
-    setup: Install .coafile via package_data
+    Diff.py: Remove has_changes and fix __bool__
 
-    When installing the .coafile to distutils.sysconfig.get_python_lib, we
-    ignore that this is not the installation directory in every case. Thus
-    it is easier, more reliable and platform independent to let distutils
-    install it by itself.
+    Removes the self.has_changes property, since its functionality can be
+    accessed from the bool conversion.
+    Fixes inconstency of __bool__ that results from looking at
+    self._changes:
+    removing one line, then adding the same content again resulted in
+    bool(diff) == True, instead of False.
 
-    Fixes https://github.com/coala/coala/issues/269
+    __bool__ now uses the mechanism that was employed by has_changes, to
+    fix this bug.
+
+    Closes https://github.com/coala/coala/issues/4178
 
 Example 2 (implemented feature):
+Pull request `#435 <https://github.com/coala/projects/pull/435>`_
 
 ::
 
-    Linter: Output command on debug
+    Update the CI1, CI2 , & CI3 tasks to refer to 2017
 
-    This massively helps debugging linters.
+    This commit changes all occurrences of 2016 to 2017 and the project
+    links with the new ones in use_coala.md, use_coala_2.md and
+    use_coala_3.md.
 
-    Closes https://github.com/coala/coala/issues/2060
+    Closes https://github.com/coala/projects/issues/433
 
 Example 3 (fixed typo):
+Pull request `#544 <https://github.com/coala/coala/pull/5544>`_
```

Comment:
`#544` -> `#5544`

Reply:
@jayvdb - Thanks for catching that. I'm also trying to add this thing you said "You could then mention that this is an example where the shortlog can use "fix", which isnt a "Fixes" because nothing was broken and it didnt have a bug label. Or maybe it is too confusing ;-)"